mava/components/building/best_checkpointer.py

```diff
@@ -30,19 +42,42 @@ def on_building_init(self, builder: SystemBuilder) -> None:
 
     def on_building_executor_start(self, builder: SystemBuilder) -> None:
         """Initialises the store for best model checkpointing"""
-        if not (builder.store.is_evaluator and self.config.checkpoint_best_perf):
+        if not (
+            builder.store.is_evaluator
+            and (self.config.checkpoint_best_perf or self.config.absolute_metric)
+        ):
             return
 
+        if self.config.absolute_metric_duration is None:
+            self.config.absolute_metric_duration = (
+                10
+                * builder.store.global_config.evaluation_duration["evaluator_episodes"]
+            )
```

Comment:
to make this a bit easier to read on the line above just do
`episodes = builder.store.global_config.evaluation_duration["evaluator_episodes"]`
and then use the episode variable.

Reply:
Will do thar!

Reply:
Done!